### ISSUE TITLE
Options for Dynamic port choice for KBS, RVPS and AS server

### ISSUE THREAD

**Issue by HarshitaBadiyasar** [confidential-containers/trustee#574]
Currently KBS server runs on 8080 port which is very common.
Is there a way to dynamically change the ports for KBS, RVPS, and AS servers?

**Comment by Xynnn007** [comment]
Yes we can. You can edit the launch configuration file/commandline to specify any ports you can.

The following are the configs used by the [docker-compose.yml](https://github.com/confidential-containers/trustee/blob/main/docker-compose.yml)

- KBS: https://github.com/confidential-containers/trustee/blob/main/kbs/config/docker-compose/kbs-config.toml#L2
- AS: https://github.com/confidential-containers/trustee/blob/main/docker-compose.yml#L38
- RVPS: https://github.com/confidential-containers/trustee/blob/main/kbs/config/rvps.json#L2
